config/search/config.json

```diff
@@ -81,6 +81,13 @@
           "http://data.vlaanderen.be/ns/besluit#bestuurt",
           "http://mu.semte.ch/vocabularies/core/uuid"
         ],
+        "administrative_unit_classification": [
```

Comment:
**Blocker:** Do we need it? If yes, you should add the `abstract_` version too

Reply:
Oh, there is no abstract_administrative_unit in the rest of the config. So I didnt add the classification either. 
And not at the moment, no. But I'm guessing we might soon as Senne is working on adding filters on classification.

Sadly I can't test locally (definitely something I need to look into more in the future). So I tested it on dev, which worked.

Reply:
It's a mistake, we can actually remove them.